### PR TITLE
Added display Initialization Function

### PR DESCRIPTION
**Pull request by Shad0w-Dragon**
-Implements TM1637TinyDisplay library
-Plays Startup Animation, then
-Sets all displays to their zero values

### CODE REVIEW

File: InverterApp/src/gpioHandler/gpioHandler.h [hunk 1]
```diff
@@ -3,6 +3,10 @@
 #include <Arduino.h>
 
 
+#define DISPLAYS_ENABLED
+
+#define SPD_DISPLAY_CLK 0
```
Comment (brennancoslett) [inline InverterApp/src/gpioHandler/gpioHandler.h:8]: Can you define all of the PINS like this?

Something like
```suggestion
## Speed Display Pins
#define SPD_CLK 0
#define SPD_DATA 1

## Coolant Temp Pins
#define Coolant_Clk 2
#define Coolant_Data 3

...
```

File: InverterApp/src/gpioHandler/gpioHandler.cpp [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
+#include "TM1637TinyDisplay.h"
 uint16_t speed = 0;
 
+
 bool GPIOHandler::Init()
 {
     // Configure all of our GPIOs
-    return true;
+    #ifdef DISPLAYS_ENABLED
+        // ALL OF OUR CODE
```
Comment (brennancoslett) [inline InverterApp/src/gpioHandler/gpioHandler.cpp:10]: remove this comment

**Review by brennancoslett — CHANGES_REQUESTED** [review]
Can you add functions for updating each of the displays that can be called elsewhere?

That way only the gpioHandler cares about the which display is which

**Comment by Shad0w-Dragon** [comment]
Defined display pins in gpioHandler.h

**Review by brennancoslett — APPROVED** [review]

**Pull request merged**